libkbfs/connection_status.go

```diff
@@ -10,10 +10,18 @@ import (
 
 // Service names used in ConnectionStatus.
 const (
-	KeybaseServiceName = "keybase-service"
-	MDServiceName      = "md-server"
+	KeybaseServiceName     = "keybase-service"
+	MDServiceName          = "md-server"
+	LoginStatusUpdateName  = "login"
+	LogoutStatusUpdateName = "logout"
 )
 
+type alwaysPushConnectionStatus struct{}
```

Comment:
Seems to be a semi-official way of doing it since it's supported by `golint`.  I agree the non-constness is annoying.  Either way is ok with me.  Usually in this library we use the `errors.New` way for unexported errors that need a special type, but I'm not sure we're 100% consistent.